examples/rust_pubsub/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "rust_pub_sub"
```

Comment:
The name of the package in `package.xml` is `rust_pubsub`, please update it here as well.

Reply:
done